python/cudf_polars/tests/experimental/test_select.py

```diff
@@ -48,7 +50,26 @@ def test_select_reduce_raises(df, engine):
         assert_gpu_result_equal(query, engine=engine)
 
 
+@pytest.mark.parametrize(
+    "aggs",
+    [
+        (
+            (pl.col("a") + pl.col("b")).sum(),
+            (pl.col("a") * 2 + pl.col("b")).alias("d").min(),
+        ),
+        (pl.col("a").min() + pl.col("b").max(),),
+        (pl.col("a") - (pl.col("b") + pl.col("c").max()).sum(),),
+        (pl.col("b").len(),),
+        (pl.col("a") - (pl.col("b") + pl.col("c").max()).mean(),),
+    ],
+)
+def test_select_aggs(df, engine, aggs):
+    # Test supported aggs (e.g. "min", "max", "std")
+    query = df.select(*aggs)
+    assert_gpu_result_equal(query, engine=engine)
+
+
 def test_select_with_cse_no_agg(df, engine):
     expr = pl.col("a") + pl.col("a")
-    query = df.select(expr, (expr * 2).alias("b"), ((expr * 2) + 10).alias("c"))
+    query = df.select(expr, (expr * 2).alias("b"), ((expr * 2) + 10).alias("d"))
```

Comment:
Why this change?

Reply:
We added a new column to the df used for testing, but this change is indeed unnecessary (reverted).